uploadhandlers/ezopenofficeuploadhandler.php

```diff
@@ -39,9 +39,9 @@
 
 class eZOpenofficeUploadHandler extends eZContentUploadHandler
 {
-    function eZOpenofficeUploadHandler()
+    function __construct()
     {
-        $this->eZContentUploadHandler( 'OOo file handling', 'openoffice' );
+        parent::__construct( 'OOo file handling', 'openoffice' );
```

Comment:
1. This change will break compatibility with eZ Publish 5.3 and 5.4 afaik.
Either we need a method_exists check, or we need to keep on calling the old one and that will work again as of next legacy release if this gets accepted: https://github.com/ezsystems/ezpublish-legacy/pull/1392

2. See the  link above, as this will for simplicity be backported to a 5.3.x release of this extension, we need BC methods for constructor changes here.
See also this one as an example: https://github.com/ezsystems/ezmbpaex/pull/10